tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs

```diff
@@ -165,6 +167,42 @@ public async Task WrittingNullStoreItems_Throws()
                 await Assert.ThrowsExceptionAsync<ArgumentNullException>(() => _storage.WriteAsync(null));
             }
         }
+        // NOTE: THESE TESTS REQUIRE THAT THE COSMOS DB EMULATOR IS INSTALLED AND STARTED !!!!!!!!!!!!!!!!!
+        [TestMethod]
+        // For issue 871
+        public async Task WaterfallCosmos()
```

Comment:
Can you a comment as to "What" this is testing? 

Looking at the tests, it's the same as all our other tests and not adding value. If it's testing something specific, please capture that knowledge in a comment. 

... and please make the "871" an actual link.

Reply:
Done, any idea why the build is failing?  It builds and all tests pass for both Dave and I locally.